### PR TITLE
Redesign invite + add member

### PR DESCRIPTION
**Pull request by antx1vmodev**

### CODE REVIEW

File: app/src/main/java/com/clearkeep/db/clear_keep/model/ChatGroup.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ data class ChatGroup(
         @ColumnInfo(name = "updated_at") val updateAt: Long,
         @ColumnInfo(name = "group_rtc_token") val rtcToken: String,
 
-        @ColumnInfo(name = "lst_client") val clientList: List<User>,
+        @ColumnInfo(name = "lst_client") var clientList: List<User>,
```
Comment (daipv92) [inline app/src/main/java/com/clearkeep/db/clear_keep/model/ChatGroup.kt:26]: change to "val"

File: app/src/main/java/com/clearkeep/screen/chat/group_create/CreateGroupActivity.kt [hunk 2]
```diff
@@ -49,7 +51,9 @@ class CreateGroupActivity : AppCompatActivity() {
                     composable("invite_group") {
                         inviteGroupViewModel.updateContactList()
                         InviteGroupScreen(
+                            InviteMember,
                             inviteGroupViewModel,
+                            listMemberInGroup= listOf<User>(),
```
Comment (daipv92) [inline app/src/main/java/com/clearkeep/screen/chat/group_create/CreateGroupActivity.kt:56]: remove this. and add listOf<User>() to InviteGroupScreen

File: app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupScreen.kt [hunk 3]
```diff
@@ -139,4 +249,9 @@ fun SelectedHorizontalBox(selectedItem: List<User>, unSelectItem: (people: User)
             SelectedFriendBox(people = friend, onRemove = { unSelectItem(it) })
         }
     }
-}
\ No newline at end of file
+}
+
+sealed class ManagerMember
```
Comment (daipv92) [inline app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupScreen.kt:254]: ManagerMember is confuse. MemberUIType is OK?

File: app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupViewModel.kt [hunk 4]
```diff
@@ -23,8 +23,8 @@ class InviteGroupViewModel @Inject constructor(
 
         val filterFriends = liveData<List<User>> {
                 val result = MediatorLiveData<List<User>>()
-                result.addSource(friends) { friendList ->
-                        result.value = getFilterFriends(friendList ?: emptyList(), textSearch.value ?: "")
+                result.addSource(friends) { _ ->
+                   //     result.value = getFilterFriends(friendList ?: emptyList(), textSearch.value ?: "")
```
Comment (daipv92) [inline app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupViewModel.kt:27]: Why comment out? is ok?

File: app/src/main/java/com/clearkeep/screen/chat/repo/GroupRepository.kt [hunk 5]
```diff
@@ -107,21 +108,36 @@ class GroupRepository @Inject constructor(
...
-
+            val response = dynamicAPIProvider.provideGroupBlockingStub().addMember(request)
+            /*
+            * update to databas
+            * */
+            if (response.success) {
+                val group = groupDAO.getGroupById(groupId, getDomain(), getClientId())
+                val newListClientId = group?.clientList?.toMutableList()
+                newListClientId?.add(
+                    User(
```
Comment (daipv92) [inline app/src/main/java/com/clearkeep/screen/chat/repo/GroupRepository.kt:127]: Add member is not return group?
should return and only update group

File: app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupViewModel.kt (hunk 4 above)
Comment (antx1vmodev) [reply app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupViewModel.kt:27]: it ok ,  Show only when the user searches

File: app/src/main/java/com/clearkeep/screen/chat/group_create/CreateGroupActivity.kt (hunk 2 above)
Comment (antx1vmodev) [reply app/src/main/java/com/clearkeep/screen/chat/group_create/CreateGroupActivity.kt:56]: fixed

File: app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupScreen.kt (hunk 3 above)
Comment (antx1vmodev) [reply app/src/main/java/com/clearkeep/screen/chat/group_invite/InviteGroupScreen.kt:254]: ok fixed

**Pull request merged**